--- DanceDanceRevolutionVGALeonardoKeyboardInput.ino
+++ DanceDanceRevolutionVGALeonardoKeyboardInput.ino
@@ -12,23 +12,18 @@
   uint8_t inputCharacterP2;
 }Pad;

-<<<<<<< HEAD
 const int SWITCH_PIN = 6;
 const int P1_LED_PIN = 7;
 const int P2_LED_PIN = 8;

 //declaring button pins
 Pad pads[] = {
   {2, HIGH, KEY_LEFT_ARROW, 'a'},
   {3, HIGH, KEY_DOWN_ARROW, 's'},
   {4, HIGH, KEY_RIGHT_ARROW, 'd'},
   {5, HIGH, KEY_UP_ARROW, 'w'}
 };

-=======
-//declaring button pins
-Pad pads[] = {{2,HIGH,'a'},{3,HIGH,'s'},{4,HIGH,'d'},{5,HIGH,'w'}};
->>>>>>> origin/main
 const int padLength = sizeof(pads) / sizeof(pads[0]);
